router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala

```diff
@@ -33,7 +33,7 @@ object TracingFilter extends SimpleFilter[Request, Response] {
     Trace.recordBinary("http.rsp.version", rsp.version.toString)
     rsp.contentLength.foreach(Trace.recordBinary("http.rsp.content-length", _))
     if (rsp.status.code >= 500 && rsp.status.code < 600) {
-      Trace.recordBinary("http.rsp.body", rsp.content.slice(0, 64 * 1000 /*64kb*/))
+      Trace.recordBinary("http.rsp.body", rsp.content.slice(0, 64 * 1000 /*64kb*/ ))
```

Comment:
hm, not related to your change, but i'm very dubious about this feature in general. what if response is chunked? What if response contains sensitive data? I'd be more in favor of checking specific headers...